config/fgcmBuildStars.py

```diff
@@ -1,10 +1,5 @@
-"""
-HSC-specific overrides for FgcmBuildStars
-"""
-
```

Comment:
Why did this get removed?  (no big deal, just wondering as in other files you just updated the Subaru part)

Reply:
These comments seemed redundant following the updates on this ticket, so I removed them where I noticed them. I wouldn't object to keeping them in however, or searching for them all and removing them all either. Keen to hear your thoughts.

Reply:
I’m a big fan of consistency, but that’s not on you as the configs are a total mix ‘n match.    You did make some deliberate choices to just update Subaru -> HSC in other files, so I was wondering if there was a specific reason to do that sometimes, but delete others.  Really not a big deal though...!!

Reply:
I agree. To be consistent, remaining vestigial references to these being HSC-specific config overrides have been removed.